tests/frontend/unit/__mocks__/layer_json.mock.js

```diff
@@ -0,0 +1,621 @@
+      isVisible: true,
+      hasDefaultVisibility: true,
+      parentId: '1b6d38af-4225-11e8-b9d0-782bcb0d78b1'
+    },
+    relationships: { categories: { data: [] }, gisLayers: { data: [] } }
+  }, {
+    type: 'GisLayerCategory',
+    id: '77c9fba8-dc29-11e8-884a-782bcb0d78b1',
+    attributes: {
+      name: 'TEST B (layerWithChildrenHidden: true) EDITED',
+      layerWithChildrenHidden: true,
+      treeOrder: 106,
+      isVisible: true,
+      hasDefaultVisibility: true,
+      parentId: '1b6d38af-4225-11e8-b9d0-782bcb0d78b1'
+    },
+    relationships: { categories: { data: [] }, gisLayers: { data: [] } }
+  }, {
+    type: 'GisLayerCategory',
+    id: '929d6a36-dc1e-11e8-884a-782bcb0d78b1',
+    attributes: {
+      name: 'Test gruppe -\u003E soll kinder verstecken',
+      layerWithChildrenHidden: true,
+      treeOrder: 107,
+      isVisible: false,
+      hasDefaultVisibility: false,
+      parentId: '1b6d38af-4225-11e8-b9d0-782bcb0d78b1'
+    },
+    relationships: { categories: { data: [] }, gisLayers: { data: [] } }
+  }, {
+    type: 'GisLayerCategory',
+    id: 'ba83fe31-dc28-11e8-884a-782bcb0d78b1',
+    attributes: {
+      name: 'ielevl',
+      layerWithChildrenHidden: false,
+      treeOrder: 108,
+      isVisible: true,
+      hasDefaultVisibility: true,
+      parentId: '1b6d38af-4225-11e8-b9d0-782bcb0d78b1'
+    },
+    relationships: {
+      categories: {
+        data: [{
+          type: 'GisLayerCategory',
+          id: '7b614b15-dc3c-11e8-884a-782bcb0d78b1'
+        }]
+      },
+      gisLayers: { data: [] }
+    }
+  }, {
+    type: 'GisLayerCategory',
+    id: 'd608e785-47a5-11e8-b9d0-782bcb0d78b1',
+    attributes: {
+      name: '33333333333333',
+      layerWithChildrenHidden: false,
+      treeOrder: 110,
+      isVisible: true,
+      hasDefaultVisibility: true,
+      parentId: '1b6d38af-4225-11e8-b9d0-782bcb0d78b1'
+    },
+    relationships: { categories: { data: [] }, gisLayers: { data: [] } }
+  }, {
+    type: 'GisLayerCategory',
+    id: 'd7636f26-4f9d-11e8-ba0c-782bcb0d78b1',
+    attributes: {
+      name: 'ööööööö',
+      layerWithChildrenHidden: false,
+      treeOrder: 109,
+      isVisible: true,
+      hasDefaultVisibility: true,
+      parentId: '1b6d38af-4225-11e8-b9d0-782bcb0d78b1'
+    },
+    relationships: { categories: { data: [] }, gisLayers: { data: [] } }
+  }, {
+    type: 'GisLayer',
+    id: '4586ce71-f68a-11e5-b083-005056ae0004',
+    attributes: {
+      hasDefaultVisibility: true,
+      isBplan: false,
+      isPrint: false,
+      isEnabled: true,
+      isXplan: false,
+      legend: '',
+      layers: 'stadtplan',
+      name: 'Stadtkarte ORKa',
+      mapOrder: 3,
+      opacity: 100,
+      procedureId: '45752f51-f68a-11e5-b083-005056ae0004',
+      serviceType: 'wms',
+      type: 'base',
+      isBaseLayer: true,
+      tileMatrixSet: null,
+      url: 'http://geo.sv.rostock.de/geodienste/stadtplan/wms?service=WMS&version=1.1.1&request=GetMap',
```

Comment:
Is it ok to use these layers?

Reply:
Thanks for pointing that out. I replaces all of them with the basemap
5813dad658afc1f94a46a8c1dfbc3470d7ace597